mltsp/ext/rethinkdb_backend.py

```diff
@@ -0,0 +1,232 @@
+# -*- coding: utf-8 -*-
```

Comment:
Where does this file come from?  If it is imported from elsewhere, we need to mind the license and annotate it correctly.